test_data/makedata_suites/005_check_system_collection.js

```diff
@@ -26,46 +31,34 @@
       print(`checking per database data ${dbCount}`);
-      let baseName = database;
-      if (baseName === "_system") {
-        baseName = "system";
-      }
-      const databaseName = `${baseName}_${dbCount}${dbNameSuffix}`;
+      const dbNameSuffix = "_testPregelSysCol";
       const pregelSystemCollectionName = '_pregel_queries';
-      let c = zeroPad(dbCount + options.countOffset);
-      db._useDatabase(databaseName);
+      {
+        print(`Checking availablitty of system collection: ${pregelSystemCollectionName} in database _system`);
+        db._useDatabase("_system");
+        assertTrue(db[pregelSystemCollectionName]);
+        const properties = db[pregelSystemCollectionName].properties();
+        assertTrue(properties.isSystem);
+        print(`Verified availability of system collection ${pregelSystemCollectionName} in database _system`);
+      }
 
-      print(`Checking availablitty of system collection: ${pregelSystemCollectionName} in database ${databaseName}`);
-      assertTrue(db[pregelSystemCollectionName]);
-      const properties = db[pregelSystemCollectionName].properties();
-      assertTrue(properties.isSystem);
-      print(`Verified availability of system collection ${pregelSystemCollectionName} in database ${databaseName}`);
-      return 0;
-    },
-    checkData: function (options, isCluster, isEnterprise, dbCount, loopCount, readOnly) {
-      if(!(semver.gt(db._version(), "3.11.0"))){
-        print("Current version doesn't support pregel system collection. Skipping test.");
-        return;
+      {
+        let baseName = database;
+        if (baseName === "_system") {
+          baseName = "system";
+        }
+        const databaseName = `${baseName}_${dbCount}${dbNameSuffix}`;
+        let c = zeroPad(dbCount + options.countOffset);
+        db._useDatabase(databaseName);
+        print(`Checking availablitty of system collection: ${pregelSystemCollectionName} in database ${databaseName}`);
+        assertTrue(db[pregelSystemCollectionName]);
+        const properties = db[pregelSystemCollectionName].properties();
+        assertTrue(properties.isSystem);
+        print(`Verified availability of system collection ${pregelSystemCollectionName} in database ${databaseName}`);
       }
-      const pregelSystemCollectionName = '_pregel_queries';
-      let databaseName = "_system";
-      db._useDatabase(databaseName);
-      print(`Checking availablitty of system collection: ${pregelSystemCollectionName} in database ${databaseName}`);
-      assertTrue(db[pregelSystemCollectionName]);
-      const properties = db[pregelSystemCollectionName].properties();
-      assertTrue(properties.isSystem);
-      print(`Verified availability of system collection ${pregelSystemCollectionName} in database ${databaseName}`);
-      return;
+      return 0;
     },
     clearDataDB: function (options, isCluster, isEnterprise, database, dbCount) {
       const dbNameSuffix = "_testPregelSysCol";
```

Comment:
do you actually create a database that needs dropping here? 
I thought we only watch in this?